### PR TITLE
Apply redesign to org settings area

### PR DESCRIPTION
**Pull request by eseliger**
| **Before** | **After** |
| --- | --- |
| image | image | 
| image |  image | 
| image | image |

### CODE REVIEW

**Review by rrhyne — APPROVED** [review]
Looks great.

File: client/web/src/org/settings/members/InviteForm.tsx [hunk 1]
```diff
@@ -102,78 +102,67 @@ export const InviteForm: React.FunctionComponent<Props> = ({
...
+                                    <AddIcon className="icon-inline" />
+                                )}{' '}
+                                Add member
+                            </button>
+                        )}
+                        {(emailInvitesEnabled || !viewerCanAddUserToOrganization) && (
+                            <button
+                                type={viewerCanAddUserToOrganization ? 'button' : 'submit'}
+                                disabled={loading === 'addUserToOrganization' || loading === 'inviteUserToOrganization'}
+                                className={`btn ${viewerCanAddUserToOrganization ? 'btn-secondary' : 'btn-primary'}`}
```
Comment (valerybugakov) [inline client/web/src/org/settings/members/InviteForm.tsx:144]:
```suggestion
                                className={classNames('btn', viewerCanAddUserToOrganization ? 'btn-secondary' : 'btn-primary')}
```

**Review by valerybugakov — APPROVED** [review]
Turbo redesign 🚀

Do we need this margin after the number of members?

Screenshot 2021-06-01 at 16 20 00

Also, it might good to add a small margin to the container to render some space after it if the list of members is bigger than the viewport height.

**Comment by eseliger** [comment]
> Do we need this margin after the number of members?

No! This PR will fix that https://github.com/sourcegraph/sourcegraph/pull/21563 :)

**Pull request merged**